config/test.exs

```diff
@@ -4,7 +4,7 @@ import Config
 # you can enable the server option below.
 config :livebook, LivebookWeb.Endpoint,
   http: [port: 4002],
-  server: false
+  server: true
```

Comment:
Let's use `get(conn, ...)` in tests instead of Bandit so we don't need this nor the next changes in application.ex.